testcafe/12_previsualisation.js

```diff
@@ -0,0 +1,30 @@
+import { Selector } from 'testcafe'
+import createUserRoleFromUserSandbox from './helpers/createUserRoleFromUserSandbox'
+import { fetchSandbox } from './helpers/sandboxes'
+import { ROOT_PATH } from '../src/utils/config'
+
+fixture('Quand je suis un utilisateur pro, et que je previsualise une de mes offres').beforeEach(
+  async t => {
+    const userRole = await createUserRoleFromUserSandbox(
+      'webapp_12_previsualisation',
+      'get_existing_pro_validated_user_with_at_least_one_visible_activated_offer'
+    )
+    await t.useRole(userRole)
+  }
+)
+
+test('je vois mon offre prévisualisé dans le carrousel', async t => {
```

Comment:
```suggestion
test('je vois mon offre prévisualisé', async t => {
```
Il y a une route dédiée pour ça `/offre`, ce n'est plus le carrousel.